webpack/redux/reducers/RedHatRepositories/sets.fixtures.js

```diff
@@ -137,6 +137,7 @@ export const successState = Immutable({
 });
 
 export const errorState = Immutable({
+  error: { response: { data: { error: { missing_permissions: ['unknown'] } } } },
```

Comment:
Would it be possible to use camel case here?
it can be confusing to have both `missing_permissions` and `missingPermissions` in the same state.

Reply:
@laviro The snake case here is just because that's how Rails gives it to us. Is there a best practice in general for exactly where in the React world it should make the jump from snake case to camel case?  I see snake case variables in JS all the time and agree that it's messy.

Reply:
usually I would prefer to rename it while receiving it from the API if possible.
Otherwise it will fastly spread to other areas of the JS code.

Reply:
usually I would prefer to rename it while receiving it from the API if possible.
Otherwise it will spread fast into other areas of the JS code.